certbot/main.py

```diff
@@ -908,11 +909,15 @@ def enhance(config, plugins):
         if not domains:
             raise errors.Error("User cancelled the domain selection. No domains "
                                "defined, exiting.")
+
+    lineage = cert_manager.lineage_for_certname(config, config.certname)
```

Comment:
Oh Python. Nice catch moving this outside of the `if not config.chain_path` branch to avoid an ugly crash down below when calling `enhancements.enable`.